core-bundle/assets/controllers/operations-menu-controller.js

```diff
@@ -31,6 +31,10 @@ export default class OperationsMenuController extends Controller {
     }
 
     disconnect() {
+        if (window.AccessibleMenu === undefined) {
+            return;
+        }
+
         // Cleanup menu instance, otherwise we would leak memory
         for (const [key, value] of Object.entries(window.AccessibleMenu.menus)) {
```

Comment:
```suggestion
        // Cleanup menu instance, otherwise we would leak memory
        for (const [key, value] of Object.entries(window.AccessibleMenu?.menus ?? {})) {
```